plugins/telemetry-otel/src/main/java/org/opensearch/telemetry/tracing/OTelTracingTelemetry.java

```diff
@@ -15,13 +15,16 @@
 
 import java.io.Closeable;
 import java.io.IOException;
+import org.opensearch.telemetry.tracing.attributes.Attributes;
+import org.opensearch.telemetry.tracing.exporter.OTelAttributesConverter;
 
 /**
  * OTel based Telemetry provider
  */
 public class OTelTracingTelemetry implements TracingTelemetry {
 
     private static final Logger logger = LogManager.getLogger(OTelTracingTelemetry.class);
+    private static final OTelAttributesConverter OTEL_ATTRIBUTES_CONVERTER = new OTelAttributesConverter();
```

Comment:
The `OTelAttributesConverter` has no state to bear, we could make it a typical utility class with static method

Reply:
Yes, I also thought of initially but though of that in future if we need to expand the converters then it would be easy. We can change it to static for now.

Reply:
One thing converter logic would be hard to test with static util method. Though I haven't added test case for now :)

Reply:
Could you add one please?

Reply:
Added the test cases and didn't make it utility class.

Reply:
> Added the test cases and didn't make it utility class.

Why? We should not create class instances when it is not needed at all

Reply:
My bad got confused with another comment. pushing it.